### DISCUSSION TITLE
Video recognition issues

### DISCUSSION THREAD

**Discussion by clareclever** [nextcloud/recognize#776] [Q&A] [answered]
How do you use the video recognition? I am running docker AIO version of Nextcloud and have the recognize app installed. I have it classifying images, but it doesn't seem to allow me to turn on video recognition. How can that setting be turned on?

**Comment by clareclever** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/59885439/228697474-11094e26-3fc0-46e5-a58d-2fe5c8fe521a.png)

I know that it's hard to show this in the image but it won't let me select the option to turn it on. It's just greyed out.

**Reply by marcelklehr** [reply]
In AIO, you probably are running in WASM mode. That doesn't support Video, IIRC

**Reply by clareclever** [reply]
I should turn this off?

If WASM mode is something that is power/GPU intensive my home server isn't going to be able to do it anyway.

![image](https://user-images.githubusercontent.com/59885439/228827854-fb425025-8d38-463f-b681-231ede236533.png)

**Reply by marcelklehr** [reply]
No, WASM mode means that your environment cannot run tensorflow natively, so recognize uses tensorflow in WASM. But that also means you cannot run video recognition.

**Reply by marcelklehr** [reply]
Turning off WASM mode will likely break all other recognition pipelines
